### DISCUSSION TITLE
OSQuery Install - Windows

### DISCUSSION THREAD

**Discussion by SasquatchSecurity** [Security-Onion-Solutions/securityonion#6578] [Unsupported Versions] [answered]
I have a PoC of Security Onion running and would like to get osquery / Fleet working for both live queries and log shipping.

I successfully got osquery installed and it's able to be queried, but data from the various Windows Event logs doesn't seem to be making it into Elastic for querying via Hunt / Kibana. I did not modify the default launcher.flags file and I can query results in the Fleet console using:
```
SELECT * FROM windows_events
```

Thanks in advance.

**Comment by SasquatchSecurity** [comment]
Modified my original request: now can't get log shipping to work properly. I want to avoid multiple agents on my hosts, so osquery seemed the best option.

**Comment by defensivedepth** [comment, marked as answer]
Can you please confirm that you set it up as follows as detailed here?:

https://docs.securityonion.net/en/2.3/osquery.html#shipping-windows-eventlogs

**Reply by SasquatchSecurity** [reply]
I did eventually find that (I missed it the first few times I was on the osquery page). I eventually decided that winlogbeat is a much better log shipper than osquery, so I'm running both. Thank you for the quick response and help!
